### DISCUSSION TITLE
Scholarship - Login and pass to farm

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#1296] [Ideas]
Could you please add a way to scholarship? Initially add login and pass and block some transactions in game.

- Manager could apply different permissions only with the owner's wallet as:
☒ Shop
☐ Craft
☒  Kitchen
---Goblin---
☐ Farm
☒ Wish
☐ Tailor
☐ Craft

**Comment by future2back** [comment]
In a future add farm renting:

**Share resource**
Time: [3] [weeks]
Share: [50%]

**Fixed fee**
Time: [1] [day]
Fee: [5] SFL

**Reply by veenhdek** [reply]
The design of the game is intentionally to not require scholarship as there is a low mint cost to start a farm and from there everything you need can be earned in game. My opinion is there is no need for a scholarship model, renting maybe if implemented properly but no need for scholars imo.
